lf-edge/edge-home-orchestration-go
# Folder Changes and MNEDC Config to YAML

Signed-off-by: Sunchit Sharma <sun.sharma@samsung.com>

# Description

1. "samples" Folder had been removed and 2 new folders : "configs" and "examples" have been created.
2. "datastorage" and "mnedc" configuration moved to configs and Native examples moved to examples
3. client.config ->client-config.yaml , corresponding code changes have been made 

Fixes #209 

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

1. Set Up one system in Main NAT as MNEDC server
2. Set up 2 system in Sub NAT as MNEDC Client
3. Checked the logs for connection
4. Requested Service from Sub NAT system to Main NAT system
5. Worked Properly as expected.


**Test Configuration**:

- Firmware version: Ubuntu 16.04
- Hardware: x86-64
- Edge Orchestration Release: Coconut

# Checklist:
